modules/nf-core/propr/logratio/main.nf

```diff
@@ -0,0 +1,23 @@
+process PROPR_LOGRATIO {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
Won't it need to be better to set higher default resources for the process? e.g.:
```suggestion
    label 'process_medium'
```

Reply:
But I don't want to allocate too many cpus for it. Should I create a custom label or should I declare the resources in the test/nextflow.config?

Reply:
The idea will be that you should allocate a sensible number of resources for running the module in a real case scenario. For module tests the resources are already limited by [this](https://github.com/nf-core/modules/blob/36cfa4c2a7e538832c50f40cb16d159e84cd3305/tests/config/nextflow.config#L8-L12) as they are also limited in the `test.config` for pipelines tests e.g. https://github.com/nf-core/differentialabundance/blob/3a849f046990707cad2a0df751c371ac7f1165ab/conf/test.config#L21-L23